### ISSUE TITLE
Building package using debug profile

### ISSUE THREAD

**Issue by jszwedko** [mmstick/cargo-deb#184]
Is it possible to change the profile that `cargo deb` uses from `release`? We build packages in CI as part of some correctness tests and would like to improve their build times by just using the debug build profile.

I attempted to `export PROFILE=debug`, but `cargo deb` still seems to build using `release`.

**Comment by kornelski** [comment]
It uses release profile only. There's a flag to skip building completely, but it will look for binaries in target/release anyway.

Maybe you can change opt-level of the release profile?

**Comment by jszwedko** [comment]
Yeah, that's a reasonable work-around. Thanks for suggesting @kornelski
